content/blog/2016/2016-05-12-sf-jam-jenkins-and-azure.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "SF JAM Report: Scaling Jenkins for Continuous Delivery with Azure"
+tags:
+- jam
+- azure
+- meetup
+author: rtyler
+---
+
+A few weeks ago, my colleague link:https://github.com/brianvdawson[Brian Dawson]
+and I were invited to present on
+link:http://www.meetup.com/jenkinsmeetup/events/228524373/[Scaling Jenkins for
+Continuous Delivery with Microsoft Azure] in Microsoft's
+link:http://microsoftreactor.com/about/[Reactor space]. Azure is Microsoft's
+public cloud offering and one of the many tools available to Jenkins users for
+adding elastic compute capacity, among other things, to their build/test/deploy
+infrastructure. While Brian's and my presentations are applicable to practically
```

Comment:
While "Brian's and my" is grammatically correct, I always go for "our" in these cases, for better readability.